server/graphql/v2/input/OrderCreateInput.ts

```diff
@@ -1,7 +1,7 @@
 import { GraphQLBoolean, GraphQLInputObjectType, GraphQLInt, GraphQLList, GraphQLNonNull } from 'graphql';
 import GraphQLJSON from 'graphql-type-json';
 
-import { ContributionFrequency } from '../enum/ContributionFrequency';
+import { ContributionFrequency } from '../enum';
```

Comment:
What's the point?

Reply:
Well, I guess not much, but my IDE gives me these warnings that the import could be shortened and I think I just clicked on it. 🙂